client/app/lib/util/checkclonedstacktemplateexist.coffee

```diff
@@ -0,0 +1,11 @@
+remote = require 'app/remote'
+
+module.exports = CheckClonedStackTemplateExist = (stackTemplate) ->
```

Comment:
This needs to be named as `camelCase` and `checkCloneExistence` might me a better name.